packages/kbn-pm/src/utils/project.js

```diff
@@ -72,9 +72,19 @@ export class Project {
     );
   }
 
+  getBuildConfig() {
+    return (this.json.kibana && this.json.kibana.build) || {};
+  }
+
   skipFromBuild() {
-    const json = this.json;
-    return json.kibana && json.kibana.build && json.kibana.build.skip === true;
+    return this.getBuildConfig().skip === true;
+  }
+
+  getCopyBuildDirectory() {
```

Comment:
`getIntermediateBuildDirectory`? With a comment along the lines of "Returns the directory that should be copied into the Kibana build artifact. This config can be specified to only include the project's build artifacts instead of everything located in the project directory."